Monitoring documentation

antimony123 opened the pull request:
- Revised monitoring and grafana docs, aggregated them into one document
- Fixed tiny change in monitoring_piman.sh to reflect recent changes in monitoring-client.py

Please note that this documentation does not cover any automation scripts which have been merged after this pull request was made. Once they have been merged, this documentation will be updated to reflect the new changes.

Commit:
Revised monitoring and grafana docs

Commit:
Minor fixes - make ip addresses generic

NickFulton98 commented:
Nice work!

bmilkov commented:
Sweet! Looks good

breed commented on monitoring/README.md:
````diff
@@ -0,0 +1,178 @@
+1. Make the /usr/bin/piman directory and copy the monitoring directory there:
+
+```
+sudo mkdir /usr/bin/piman
````
this is not a good place to put it. perhaps /usr/local/piman/bin or /usr/share/piman/bin

breed commented on monitoring/README.md:
````diff
@@ -0,0 +1,178 @@
+1. Copy the files monitoring-server.py and pi-server-monitoring.service to each pi's home directory:
+
+```
+cd /path/to/piman/monitoring
````
this was the directory above, right?

breed reviewed (changes requested):
there was a change that went in for monitoring install scripts on the pis. can you reference those instead of walking through the steps.

antimony123 committed:
Update to include pi installation scripts, and use /usr/local instead of /usr/bin as the directory for piman

antimony123 committed:
Update monitoring_piman.sh to use /usr/local instead of /usr/bin

antimony123 commented:
@breed I updated the docs to reflect the requested changes as well as the automation scripts. I also updated monitoring_piman.sh to use the better directory, /usr/local. Let me know if there is anything else to change. Thanks!

breed commented on monitoring/README.md:
````diff
@@ -37,68 +37,34 @@ The first step is to update all the pis' package repositories, install all depen
 ```
-ssh pi@<ip address>
+cd /usr/local/piman/monitoring
+sudo ./init_pis.sh pi piman /usr/local/piman/monitoring/hosts.csv /usr/local/piman/monitoring
````
this doesn't seem correct. isn't hosts.csv in /usr/local/piman?

antimony123 replied:
It is indeed. I fixed the path to be /usr/local/piman/hosts.csv.

antimony123 committed:
Fix path to hosts file

breed reviewed (approved):
looks good
